sdk/core/Azure.Core/tests/RecordSessionTests.cs

```diff
@@ -283,7 +283,7 @@ public void RecordSessionLookupSkipsRequestBodyWhenFilterIsOn()
 
             recordTransport.Process(message);
 
-            request.Content = RequestContent.Create(Encoding.UTF8.GetBytes("A bad and longer body."));
+            request.Content = RequestContent.Create(Encoding.UTF8.GetBytes("A bad and longer body"));
```

Comment:
Why did this have to change?

Reply:
Currently headers are not compared in playback mode for matching record for track 2, this is a bug, this PR is fixing the issue. Without this change, the header `Content-Length` doesn't match session file and cause #291 failure.